test/vcf/ploidy_test.cpp

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2016 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <memory>
+#include <fstream>
+
+#include "catch/catch.hpp"
+
+#include "vcf/validator.hpp"
+
+namespace ebi
+{
+
+  TEST_CASE("Validate files with combined ploidy", "[ploidy]")
+  {
+      SECTION("file with 3 as default ploidy")
```

Comment:
Please capitalize the first letter in SECTION messages as in the rest of test files.